### PR TITLE
zig 0.9.0

### PR DESCRIPTION
**Pull request by DHunte**

### CODE REVIEW

File: Food/zig.lua [hunk 1]
```diff
@@ -0,0 +1,50 @@
+local name = "zig"
+local version = "0.9.0"
+
+food = {
+    name = name,
+    description = "Zig is a general-purpose programming language and toolchain for maintaining robust, optimal, and reusable software.",
+    license = "MIT (Expat)",
```
Comment (bacongobbler) [inline Food/zig.lua:7]: You can shorten this to “MIT” - IIRC the expat variant is the one that is most commonly referred to as the MIT license. The X11 variant is the one with additional anti-advertising clauses.

**Review by bacongobbler — CHANGES_REQUESTED** [review]

File: Food/zig.lua (hunk 1 above)
Comment (DHunte) [reply Food/zig.lua]: Ah great thanks for explaining and for the info/review. I have committed the change now.

**Comment by bacongobbler** [comment]
Looks like there are syntax errors in the Lua file. Can you please verify with `gofish lint` before submitting the PR? Thanks!
